auctioneer/client.go

```diff
@@ -577,6 +577,27 @@ func (c *Client) connectAndAuthenticate(ctx context.Context,
 	c.subscribedAcctsMtx.Unlock()
 	err := sub.authenticate(ctx)
 	if err != nil {
+		log.Errorf("Authentication failed for account %x: %v",
+			acctPubKey[:], err)
+
+		// The error that's returned from authenticate() might just be
```

Comment:
Ah ok, so before this, we would just return early here, never read on the channel and then ultimately give up until a restart?

Reply:
Yes, exactly. It would just hang there. In fact if the connect was issued on startup, we'd never fully complete the server's `Start()` method and therefore hang in a state where we don't listen for shutdown signals yet. So worst case you'd even need to kill the daemon.